homeassistant/components/arcam_fmj/media_player.py

```diff
@@ -111,8 +103,12 @@ def unique_id(self):
     def device_info(self):
         """Return a device description for device registry."""
         return {
-            "identifiers": {(DOMAIN, self._state.client.host, self._state.client.port)},
-            "model": "FMJ",
+            "name": f"{DEFAULT_NAME} ({self._state.client.host})",
+            "identifiers": {
+                (DOMAIN, self._uuid),
+                (DOMAIN, self._state.client.host, self._state.client.port),
```

Comment:
host and port can change so they aren't stable, so they should likely be removed.

Reply:
Yes, we can remove it later. Once people have upgraded. Otherwise we would leave these devices unconnected to entities.